blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/PipelineSearch.java

```diff
@@ -40,6 +44,7 @@
  */
 @Extension
 public class PipelineSearch extends OmniSearch<BluePipeline>{
+    private static final String STARTSWITH_PARAM="startsWith";
```

Comment:
We do not need new param, there is already `pipeline` param, it should be used regex. Current code does equals comparison, should be using it as regex and collect all pipelines whose name match this regex 🐛